static/app/components/replays/replayPlayer.tsx

```diff
@@ -253,4 +316,40 @@ const SentryPlayerRoot = styled(PlayerRoot)`
   }
 `;
 
+const ResourceCardContainer = styled('div')`
+  position: absolute;
+  bottom: ${space(1)};
+  right: 1%;
```

Comment:
Can we make this a fixed amount? i'm gussing at a size, but 1% seems funny to me
```suggestion
  right: ${space(1)};
```

here's 1% when the container is different widths:
| wide container | narrow container |
| --- | --- |
| <img width="141" alt="SCR-20240122-lgaa" src="https://github.com/getsentry/sentry/assets/187460/3875beac-a34d-49d4-a57e-11606e076735"> | <img width="137" alt="SCR-20240122-lgaq" src="https://github.com/getsentry/sentry/assets/187460/1bd601b0-ead1-4bc9-aa9d-79a8f82fbcfd">